### ISSUE TITLE
microcode update for Xeon E3 V6 

### ISSUE THREAD

**Issue by amazingAmy** [intel/Intel-Linux-Processor-Microcode-Data-Files#41]
Is the microcode update for Xeon E3 V6 available to fix the L1TF and special register buffer data sampling?
WHAT is the latest microcode version for the Xeon E3 V6?  Is 0xd6 the latest version?

**Comment by bsdice** [comment]
Yes 0xD6 from 2020-04-23 seems to be the latest, but I recommend 0xCA because my personal Linux workstation started crashing with "illegal opcode" errors on 0xD6 with an E3-1275v6 on a Supermicro X11SAT board. Took me several days to nail the microcode as culprit down. In the end I had a "crashkernel" in place to catch the bug post-mortem, but the system always seemed too far gone for that to work. In hindsight an indication that either hardware is dying or a microcode update introduced a serious problem.

**Comment by bsdice** [comment]
Thanks for the nice reminder @hmh and yes of course I update microcode earliest possible using "initrd /intel-ucode.img" in systemd-boot, before kernel and boot ramdisk is loaded. Log message about early update is present. With D6 it takes 0-3 days and this workstation will hang so hard that it will not even kexec the crashkernel (I debugged this process with forced kernel panics - it works). The problem is gone with CA, with nothing else changed. Hence my cautionary note to Amy.

**Comment by whpenner** [comment]
Please update to the latest microcode updates.  This issue should be resolved.
